includes/specials/SpecialUserrights.php

```diff
@@ -392,7 +427,7 @@ function switchForm() {
 	 * @return Array:  Tuple of addable, then removable groups
 	 */
 	protected function splitGroups( $groups ) {
-		list( $addable, $removable, $addself, $removeself ) = array_values( $this->changeableGroups() );
+		list( $addable, $removable, $addself, $removeself ) = array_values( $this->getUser()->changeableGroups() );
```

Comment:
Any reason why you removed the method `changeableGroup()` and inlined its content in a few places in this file?